frontend/packages/console-app/src/plugin.tsx

```diff
@@ -0,0 +1,25 @@
+
+import * as React from 'react';
+import { CogIcon } from '@patternfly/react-icons';
+import {
+  Plugin,
+  Perspective,
+} from '@console/plugin-sdk';
+
+type ConsumedExtensions =
+  | Perspective;
```

Comment:
Do we want to add a `Perspective` extension example to `console-demo-plugin` too?

(I was thinking of a unit test that checks if demo plugin uses all supported extension types.)

Reply:
Added a sample perspective.